app/views/moderations/mod.html.erb

```diff
@@ -189,7 +189,7 @@
         value="<%= rating_level %>"
         name="rating_vote_<%= rating_level %>"
         id="js__rating__vote__<%= rating_level %>"
-        class="level-rating-button article-mod-page <%= "selected" if @rating_vote&.rating == rating_level.to_f %>"
+        class="level-rating-button article-mod-page <%= "selected" if @rating_vote&.rating.to_d == rating_level.to_d %>"
```

Comment:
`rating_level` is an integer, float comparison is imprecise, see https://rubystyle.guide#float-comparison